packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Prefix.Example.tsx

```diff
@@ -6,8 +6,8 @@ export class TextFieldPrefixExample extends React.Component<any, any> {
   public render(): JSX.Element {
     return (
       <div className="docs-TextFieldExample">
-        <TextField prefix="https://" />
-        <TextField label="Disabled" prefix="https://" disabled />
+        <TextField ariaLabel="Prefix https://" prefix="https://" />
```

Comment:
Maybe something like "Example text field with https:// prefix" would be better?

Reply:
Same in the other prefix/suffix example files--would be nice to have more descriptive text